### PR TITLE
Feature/PRLC100-209

### PR DESCRIPTION
**Pull request by giorgi-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRLC100-209

### Change description ###
- Add content to `c100-rebuild/contact-representative` page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/contact-representative/content.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {},
+  submit: {
+    text: () => 'Close the application',
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/contact-representative/content.ts:43]: Can we move this to variables

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/contact-representative/content.ts (hunk 1 above)
Comment (giorgi-hmcts) [reply src/main/steps/c100-rebuild/contact-representative/content.ts]: Moved inside variables, all tests passing ✅

**Review by yogendraU — APPROVED** [review]

**Review by channa-patil — CHANGES_REQUESTED** [review]
@giorgi-hmcts Can we move 'contact-representative' folder inside 
src/main/steps/c100-rebuild/screening-questions as this is part of screening questions feature. Also please append /screening-questions in the URL

**Comment by giorgi-hmcts** [comment]
> @giorgi-hmcts Can we move 'contact-representative' folder inside src/main/steps/c100-rebuild/screening-questions as this is part of screening questions feature. Also please append /screening-questions in the URL

Moved 'contact-representative' inside inside 'src/main/steps/c100-rebuild/screening-questions' and updated the URL ✅

**Pull request merged**